lightning/src/onion_message/messenger.rs

```diff
@@ -1633,7 +1644,7 @@ where
 								return
 							}
 						};
-						let response_instructions = self.offers_handler.handle_message(msg, context, responder);
+						let response_instructions = self.offers_handler.handle_message(msg, context, custom_data, responder);
```

Comment:
We need to give the data to all handlers, not just the offers handler.

Reply:
Done! [pr2830.11](https://github.com/shaavan/rust-lightning/commits/pr2830.11)
Thanks for the pointer!